### DISCUSSION TITLE
Disable console output?

### DISCUSSION THREAD

**Discussion by VirtualMachineMan** [mitmproxy/mitmproxy#5893] [Q&A] [answered]
How do I get rid of the default terminal thingy, and make it so it only output's python print()s?

Basically, hide mitmproxy but show my printed stuff.

**Comment by Prinzhorn** [comment, marked as answer]
`mitmdump --options` to see all options, `mitmdump --help` also includes it => `mitmdump --quiet`

**Comment by VirtualMachineMan** [comment]
mitmdump --quiet doesn't work, still shows the flows window.

**Reply by Prinzhorn** [reply]
That doesn't make sense, are you sure you're using `mitmdump --quiet` and not `mitmproxy --quiet`?

**Reply by VirtualMachineMan** [reply]
Ohhhh. Makes sense. Thank you.
